src/modules/job-ingest/schemas/jobspec.jsonschema

```diff
@@ -98,7 +98,7 @@
         "system": {
           "type": "object",
           "properties": {
-            "duration": { "type": "string" }
+            "duration": { "type": "number" }
```

Comment:
The schema could restrict duration to a positive number using
```json
"duration": { "type": "number", "exclusiveMinimum": 0 }
```
(I assume zero is not allowed?)

Reply:
Good point!

Actually we may need a way to communicate "unlimited" -- should `0` be allowed for that case? (wish I would have thought of this when modifying RFC 14)

Reply:
Just pushed a patch to set the minimum to `0`. I think we may have to allow `0` with special meaning -- maybe clarify this in RFC 14 if you agree?

Reply:
Sure

Reply:
flux-framework/rfc#178